<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Universal Data Share</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Local</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="localList">
            <attribute name="headerDefaultSectionSize">
             <number>160</number>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>150</number>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Size</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Path</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Remote</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="remoteAddress"/>
            </item>
            <item>
             <widget class="QPushButton" name="Goto">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="refresh">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="remoteList">
            <attribute name="headerDefaultSectionSize">
             <number>150</number>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Size</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Downloads</string>
         </attribute>
         <widget class="QTreeWidget" name="downloadsView">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>531</width>
            <height>441</height>
           </rect>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>150</number>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Progress</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Save Path</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuShares">
    <property name="title">
     <string>Shares</string>
    </property>
    <addaction name="actionCopy_Global_URL"/>
    <addaction name="actionGo_online"/>
    <addaction name="actionGo_offline"/>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>Remote</string>
    </property>
    <addaction name="actionDownload_selected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuShares"/>
   <addaction name="menuRemote"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusView"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionCopy_Global_URL">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Copy Global URL</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGo_online">
   <property name="text">
    <string>Go online</string>
   </property>
  </action>
  <action name="actionGo_offline">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go offline</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDownload_selected">
   <property name="text">
    <string>Download selected</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
